howso/hierarchy.amlg

```diff
@@ -881,12 +1052,59 @@
 				{
 					parent_id !parentId
 					path_to_id_map !containedTraineeNameToIdMap
-					id_to_path_map  !containedTraineeIdToNameMap
+					id_to_path_map !containedTraineeIdToNameMap
 					is_contained_map !childTraineeIsContainedMap
 				}
 		))
 	)
 
+	;method to manually add hierarchy references to an already loaded subtrainee.
+	;This assumes a subtrainee has already been loaded into the contained entity ".trainee_container" of this trainee.
+	;e.g., if this trainee entity id is "airplane" and a subtrainee of "right_engine" is being loaded,
+	;it should have been loaded into the entity path of: ["airplane" ".trainee_container" "right_engine"],
+	;( Warning: subtrainees must be loaded into the ".trainee_container" entity for proper functionality. )
+	;and then this method can be called on "airplane" with parameter: entity_path_id = "right_engine"
+	;
+	;{idempotent (true) }
```

Comment:
We tag `set_hierarchy_relationships` as `protected`, should this one be as well?